### PR TITLE
Fix small issues in Mvc.js

### PR DESCRIPTION
**Pull request by scottaddie**
Made the following changes to Mvc.js:
1. Enabled strict mode to prevent doing bad things in JavaScript
2. Added a missing semi-colon
3. Switched to using double quotes where single quotes were used, to enforce consistency throughout the file

### CODE REVIEW

**Comment by dougbu** [comment]
@scottaddie thank-you for the contribution. I'll have a closer look at this over the weekend or Monday.

In the meantime please ignore the CI build failures shown above. Right now our feeds are not in a state where those checks can succeed.

File: samples/MvcSample.Web/wwwroot/Scripts/Mvc.js [hunk 1]
```diff
@@ -29,7 +31,7 @@ var MVC = (function () {
             return;
         }
 
-        if (Object.prototype.toString.call(value) === '[object Array]') {
+        if (Object.prototype.toString.call(value) === "[object Array]") {
```
Comment (Kagamine) [inline samples/MvcSample.Web/wwwroot/Scripts/Mvc.js]: I think this change is no need.

Comment (scottaddie) [inline samples/MvcSample.Web/wwwroot/Scripts/Mvc.js]: It doesn't change functionality, but it does enforce consistency. I suppose it depends on what the project's coding standards are. In my mind, consistency leads to a lower barrier to entry for contributors on this project.

**Comment by dougbu** [comment]
92afcaa

**Pull request closed without merging**